### PR TITLE
RCHAIN-3964 add conf

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
Make the hard-coded number in the rnode to be configurable.
add `numberOfActiveValidators` ,  `casperLoopInterval` and `requestedBlocksTimeout`

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3964

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Review by woky — APPROVED** [review]

File: node/src/main/resources/reference.conf [hunk 1]
```diff
@@ -156,6 +156,15 @@ rnode {
...
     quarantine-length = 50000
 
+    # the number of active validator in casper
+    number-of-active-validators = 100
+
+    # the interval of the casper loop to maintain requested blocks and missing dependent blocks
+    casper-loop-interval = 30
+
+    # the timeout of the requested blocks
+    requested-blocks-timeout = 240
```
Comment (tgrospic) [inline node/src/main/resources/reference.conf:170]: Is there any description in which units are interval and timeout?

**Review by tgrospic — APPROVED** [review]
LGTM. One minor comment.

**Pull request merged**